cfme/tests/containers/test_logging.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+from utils import testgen
+from cfme.containers.provider import ContainersProvider
```

Comment:
@psav i think i order this correctly but i am not sure,
please take a look on this